lib/production_e2e_filter.rb

```diff
@@ -0,0 +1,64 @@
+require 'logger'
+
+module ProductionE2EFilter
+  def self.log_tag
+    return 'production_e2e_filter'
+  end
+
+  BEGIN {
+    logger = Logger.new(STDOUT)
+    begin
+      E2E_FILTER_CONF = YAML::load(ERB.new(IO.read(File.expand_path('./conf/e2e_filter_conf.yml', __FILE__))).result).with_indifferent_access
+      @emails = E2E_FILTER_CONF['PREFIXES']['EMAILS']
+      @domains = E2E_FILTER_CONF['PREFIXES']['DOMAINS']
+      logger.info({
+                    log_tag: self.log_tag,
+                    message: "Successfully initialized red_cross production e2e filter values"
+                  })
+    rescue => e
+      logger.error({
+                     log_tag: self.log_tag,
+                     message: "Unable to initialize red_cross production e2e filter due to #{e.message}"
+                   })
+    end
+  }
+
+  def is_filtering_enabled?
+    E2E_FILTER_CONF['EVENT_FILTERING_ENABLED']
```

Comment:
why did you take the EVENT_FILTERING_ENABLED from the yml and not from env? 
assuming you want to disable this ability - changing a yaml file will require you to push something to GIT as oppose to env vars that can be modified via jenknins redeploy operation without any code/GIT envolved.